### PR TITLE
Update errors.csv

### PR DESCRIPTION
**Pull request by x0x8x**

### CODE REVIEW

File: telethon_generator/data/errors.csv [hunk 1]
```diff
@@ -7,22 +7,22 @@ ADMINS_TOO_MUCH,400,Too many admins
 ADMIN_RANK_EMOJI_NOT_ALLOWED,400,Emoji are not allowed in admin titles or ranks
 ADMIN_RANK_INVALID,400,The given admin title or rank was invalid (possibly larger than 16 characters)
 API_ID_INVALID,400,The api_id/api_hash combination is invalid
-API_ID_PUBLISHED_FLOOD,400,"This API id was published somewhere, you can't use it now"
+API_ID_PUBLISHED_FLOOD,400,This API id was published somewhere, you can't use it now
```
Comment (Lonami) [inline telethon_generator/data/errors.csv:10]: This change is wrong, the quotes are needed or the columns will be incorrect.

File: telethon_generator/data/errors.csv [hunk 2]
```diff
@@ -7,22 +7,22 @@ ADMINS_TOO_MUCH,400,Too many admins
...
 ARTICLE_TITLE_EMPTY,400,The title of the article is empty
 AUDIO_TITLE_EMPTY,400,The title attribute of the audio must be non-empty
 AUTH_BYTES_INVALID,400,The provided authorization is invalid
-AUTH_KEY_DUPLICATED,406,"The authorization key (session file) was used under two different IP addresses simultaneously, and can no longer be used. Use the same session exclusively, or use different sessions"
+AUTH_KEY_DUPLICATED,406,The authorization key (session file) was used under two different IP addresses simultaneously, and can no longer be used. Use the same session exclusively, or use different sessions
```
Comment (Lonami) [inline telethon_generator/data/errors.csv:14]: Same as above.

File: telethon_generator/data/errors.csv [hunk 3]
```diff
@@ -7,22 +7,22 @@ ADMINS_TOO_MUCH,400,Too many admins
...
 AUTH_KEY_INVALID,401,The key is invalid
-AUTH_KEY_PERM_EMPTY,401,"The method is unavailable for temporary authorization key, not bound to permanent"
+AUTH_KEY_PERM_EMPTY,401,The method is unavailable for temporary authorization key, not bound to permanent
```
Comment (Lonami) [inline telethon_generator/data/errors.csv:16]: Same as above.

File: telethon_generator/data/errors.csv [hunk 4]
```diff
@@ -46,21 +46,21 @@ CALL_OCCUPY_FAILED,500,The call failed because the user is already making anothe
...
+CHANNEL_INVALID,400,Invalid channel object. Make sure to pass the right types, for instance making sure that the request is designed for channels or otherwise look for a different one more suited
 CHANNEL_PRIVATE,400,The channel specified is private and you lack permission to access it. Another reason may be that you were banned from it
 CHANNEL_PUBLIC_GROUP_NA,403,channel/supergroup not available
 CHAT_ABOUT_NOT_MODIFIED,400,About text has not changed
 CHAT_ABOUT_TOO_LONG,400,Chat about too long
 CHAT_ADMIN_INVITE_REQUIRED,403,You do not have the rights to do this
-CHAT_ADMIN_REQUIRED,400 403,"Chat admin privileges are required to do that in the specified chat (for example, to send a message in a channel which is not yours), or invalid permissions used for the channel or group"
-CHAT_FORBIDDEN,,You cannot write in this chat
+CHAT_ADMIN_REQUIRED,400,Chat admin privileges are required to do that in the specified chat (for example, to send a message in a channel which is not yours), or invalid permissions used for the channel or group
```
Comment (Lonami) [inline telethon_generator/data/errors.csv:57]: Are you sure this error is not a 403?

File: telethon_generator/data/errors.csv [hunk 5]
```diff
@@ -7,22 +7,22 @@ ADMINS_TOO_MUCH,400,Too many admins
...
 AUTH_KEY_INVALID,401,The key is invalid
-AUTH_KEY_PERM_EMPTY,401,"The method is unavailable for temporary authorization key, not bound to permanent"
+AUTH_KEY_PERM_EMPTY,401,The method is unavailable for temporary authorization key, not bound to permanent
 AUTH_KEY_UNREGISTERED,401,The key is not registered in the system
 AUTH_RESTART,500,Restart the authorization process
 AUTH_TOKEN_ALREADY_ACCEPTED,400,The authorization token was already used
 AUTH_TOKEN_EXPIRED,400,The provided authorization token has expired and the updated QR-code must be re-scanned
 AUTH_TOKEN_INVALID,400,An invalid authorization token was provided
-BANNED_RIGHTS_INVALID,400,"You cannot use that set of permissions in this request, i.e. restricting view_messages as a default"
+BANNED_RIGHTS_INVALID,400,You cannot use that set of permissions in this request, i.e. restricting view_messages as a default
```
Comment (Lonami) [inline telethon_generator/data/errors.csv:22]: You know the drill.

**Review by Lonami — CHANGES_REQUESTED** [review]
A lot of quotes were removed when not needed, and the only other change needs confirmation.

File: telethon_generator/data/errors.csv (hunk 4 above)
Comment (x0x8x) [reply telethon_generator/data/errors.csv:57]: Yes, I'm sure
```
{
	"_": "rpc_error",
	"error_code": 400,
	"error_message": "CHAT_ADMIN_REQUIRED"
}
```

Looks like someone is moved accidentally code `403 `from the `CHAT_FORBIDDEN` to the `CHAT_ADMIN_REQUIRED`

**Pull request closed without merging**